aten/src/ATen/native/native_functions.yaml

```diff
@@ -183,6 +183,14 @@
   dispatch:
     CompositeExplicitAutograd: _functional_assert_scalar
 
+- func: _record_scalar(Scalar self, str prefix, str filename) -> ()
+  dispatch:
+    CompositeExplicitAutograd: _record_scalar
+
+- func: _functional_record_scalar(Scalar self, str prefix, str filename, Tensor dep_token) -> Tensor
+  dispatch:
+    CompositeExplicitAutograd: _functional_record_scalar
+
```

Comment:
I don't think we need this form -- `_functional_assert_async.msg` was from a previous implementation of effect tokens that we don't use anymore